TabloidCLI/UserInterfaceManagers/JournalManager.cs

```diff
@@ -0,0 +1,160 @@
+        private string _connectionString;
+
+        public JournalManager(IUserInterfaceManager parentUI, string connectionString)
+        {
+            _parentUI = parentUI;
+            _journalRepository = new JournalRepository(connectionString);
+            _connectionString = connectionString;
+        }
+
+        public IUserInterfaceManager Execute()
+        {
+            Console.WriteLine("My Journal Management");
+            Console.WriteLine("1) View Journal Entries");
+            Console.WriteLine("2) Add Entry");
+            Console.WriteLine("3) Edit Entry");
+            Console.WriteLine("4) Remove Entry");
+            Console.WriteLine("0) Return to Main Menu");
+
+            Console.Write(">");
+            string choice = Console.ReadLine();
+            switch (choice)
+            {
+                case "1":
+                    List();
+                    return this;
+                case "2":
+                    Add();
+                    return this;
+                case "3":
+                    Edit();
+                    return this;
+                case "4":
+                    Remove();
+                    return this;
+                case "0":
+                    return _parentUI;
+                default:
+                    Console.WriteLine("Invalid Selection");
+                    return this;
+            }
+        }
+
+
+
+        private void List()
+        {
+            List<Journal> journals = _journalRepository.GetAll();
+            foreach (Journal journal in journals)
+            {
+                Console.WriteLine(journal);
+            }
+        }
+
+        private Journal Choose(string prompt = null)
+        {
+            if (prompt == null)
+            {
+                prompt = "Please choose a Journal:";
+            }
+
+            Console.WriteLine(prompt);
+
+            List<Journal> journals = _journalRepository.GetAll();
+
+            for (int i = 0; i < journals.Count; i++)
+            {
+                Journal journal = journals[i];
+                Console.WriteLine($" {i + 1}) {journal.Title}");
+            }
+            Console.Write("> ");
+
+            string input = Console.ReadLine();
+            try
+            {
+                int choice = int.Parse(input);
+                return journals[choice - 1];
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Invalid Selection");
+                return null;
+            }
+        }
+
+
+        private void Add()
+        {
+            Console.WriteLine("New Journal Entry");
+            Journal journal = new Journal();
+
+            Console.Write("Title");
+            journal.Title = Console.ReadLine();
+
+            Console.Write("Content");
+            journal.Content = Console.ReadLine();
+
+            Console.Write("Date Created");
+            journal.CreateDateTime = DateTime.Parse(Console.ReadLine());
```

Comment:
"***Then*** the new journal entry should be saved to the database with the current date and time saved as the creation date"

-->This assignment needs to be set to the current date and time, rather than user input

The program breaks here if the user enters a string/anything that isn't in the right format. If you want to keep it as user input, you might need to write a try/catch or if/else statement